config/database.yml

```diff
@@ -0,0 +1,16 @@
+default: &default
+  adapter: mysql2
+  encoding: utf8
+  pool: 5
+  username: root
+  password: Genius15?
+  socket: /var/run/mysqld/mysqld.sock
```

Comment:
you should use gem dotenv (https://github.com/bkeepers/dotenv) to write security variables such as account name, password in 'env' file, then ignore this file by '.gitignore'. Otherwise others can see them